FeedKit/images/ImageRepository.swift

```diff
@@ -262,7 +272,7 @@ extension ImageRepository {
 extension ImageRepository: Images {
 
   public func cancel(displaying view: UIImageView) {
-    Nuke.cancelRequest(for: view)
+    view.nk.cancelImageRequest()
```

Comment:
meh

Reply:
To be honest, I'm not a fan of these new prefixes either. I avoided adding them for 7 major releases. It's not too late to revert this change, that's why I ship betas. 

The motivation for finally adding them was:

- To rename `Nuke.loadImage(with: url: into: view)` because it doesn't seem to be a grammatically correct sentence
- This seems to be a standard practice across frameworks

I do like to have a bit of quirkiness in the framework though, that's why I'm keeping the name `ImagePreheater` (`ImagePrefetcher` is a more clear name). So maybe it's best to keep these APIs as is.

Reply:
I prefer *pure* functions where I’m passing a thing to extended types. Asking a view to cancel a image request is not intuitive, but I understand the intention, of course. The view is the *key*, so why not use its identity from within itself. I have never liked libraries that publicly extended types. I want a discrete API surface.

Reply:
quirkiness ftw